### DISCUSSION TITLE
How many api request can we use perday

### DISCUSSION THREAD

**Discussion by ghost** [facebookresearch/esm#382] [ESM Metagenomic Atlas]
### API request
I am a computer science researcher who needs to proceed with a dataset that consists 4000 extremely short protein sequences (generally less than 100 amino acids in length),  I tried the API using [ESM Metagenomic Atlas](https://esmatlas.com/), and it was successful for the very first 500 sequences; then I got a forbidden message for the remaining ones. The general proceed speed is approximately one iteration per second
Best regards

**Comment by tomsercu** [comment]
This is a duplicate of #375 - we'll increase rate limits for fetching data

**Reply by ghost** [reply]
thanks for your reply, I am looking forward to that, can't wait to use your amazing work to do some impressive work
